<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyMyselfDialog</class>
 <widget class="QWidget" name="ModifyMyselfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ModifyMyselfDialog{
    background-color : #FFFFFF;
    border : 1px solid #DDDDDD;
}</string>
  </property>
  <widget class="QLabel" name="tittleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tittleLabel{
    background-color : #F8F8F8;
    padding-left : 16px;
    font-family : 微软雅黑;
    font-size : 14px;
    color : #222222;
    font-weight : bold;
}</string>
   </property>
   <property name="text">
    <string>修改个人资料</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>72</y>
     <width>32</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#emailLabel{
    font-family : 微软雅黑;
    font-size : 14px;
    color : #222222;
}</string>
   </property>
   <property name="text">
    <string>邮箱:</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailNumberLabel">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>72</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>569334855@qq.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>122</y>
     <width>32</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#passwordLabel{
    font-family : 微软雅黑;
    font-size : 14px;
    color : #222222;
}</string>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="passwordBtn">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>114</y>
     <width>252</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#passwordBtn{
    background-color : #FFFFFF;
    border : 1px solid #DDDDDD;
    border-radius : 2px;
    text-align : left;
    padding-left : 10px;
    color : #999999;
    font-family : 微软雅黑;
    font-size : 14px;
}</string>
   </property>
   <property name="text">
    <string>点击设置密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="passwordWidget" native="true">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>123</y>
     <width>100</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="newPasswordLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#newPasswordLabel{
    font-family : 微软雅黑;
    font-size : 14px;
    color : #999999;
}</string>
    </property>
    <property name="text">
     <string>已设置新密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changePasswordBtn">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>0</y>
      <width>19</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#changePasswordBtn{
     border : none;
     background-color : transparent;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="iamgeRes.qrc">
      <normaloff>:/images/myself/xiugaimima.png</normaloff>:/images/myself/xiugaimima.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>12</width>
      <height>12</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="nickNameLabel">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>180</y>
     <width>32</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#nickNameLabel{
    font-family : 微软雅黑;
    font-size : 14px;
    color : #222222;
}</string>
   </property>
   <property name="text">
    <string>昵称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickNameEdit">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>172</y>
     <width>258</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#nickNameEdit{
    background-color : #FFFFFF;
    border : 1px solid #DDDDDD;
    border-radius : 2px;
    color : #999999;
    font-family : 微软雅黑;
    font-size : 14px;
    padding-left : 10px;
}</string>
   </property>
   <property name="text">
    <string>比特视频用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitBtn">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>264</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#submitBtn{
    background-color : #3ECEFE;
    color : #FFFFFF;
    border : none;
    border-radius : 2px;
    font-family : 微软雅黑;
    font-size : 14px;
}</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>264</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cancelBtn{
    background-color :transparent;
    color : #222222;
    border : 1px solid #DDDDDD;
    border-radius : 2px;
    font-family : 微软雅黑;
    font-size : 14px;
}</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="iamgeRes.qrc"/>
 </resources>
 <connections/>
</ui>
